### DISCUSSION TITLE
Disable completion when hardtime is enabled

### DISCUSSION THREAD

**Discussion by smarquez1** [m4xshen/hardtime.nvim#66] [Q&A]
I'm using this plugin while I practice algorithms. I'd like to disable completion and copilot while `hardtime` is enabled so I learn not to really on those aids.

Is there any way to do this?

**Comment by m4xshen** [comment]
Hmm, I don't think of any way to do this automatically. Maybe you can just setup a mapping to disable Copilot and enable Hardtime.
```Lua
map("n", "<Leader>a", ":Copilot disable<CR>:Hardtime enable<CR>", { desc = "[A]lgorithm Mode" })
-- change the <Leader>a to any mapping you want
```

**Reply by smarquez1** [reply]
Nice! I was thinking of adding some kind of callback support for Hardtime, but that will do.
Thanks @m4xshen
